### DISCUSSION TITLE
Welcome to iecUnitTest Discussions!

### DISCUSSION THREAD

**Discussion by tkucic** [tkucic/UniTest#333] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheColonel2688** [comment]
I am interested in using this on Codesys. Is there a full walk-through or an example project?

**Reply by tkucic** [reply]
Hi,
in the repository readme.md there is an example in codesys. The codesys version of the library has been tested by the framework, you can check that out if you open the project file in codesys.

Now, it has been some time since I worked on this library so it will probably need an update to the latest codesys version.

Let me know if you have trouble with the library or suggest improvements. I have a few of my own but, as the user base is quite low, so far it has been low priority.

**Reply by tkucic** [reply]
Btw, Codesys has already one unittest framework, quite well integrated. Its called Code https://forge.codesys.com/lib/counit/home/Home/ forked from TcUnit for TwinCat.

UniTest is mostly designed to provide minimum testing capabilities to non-codesys platforms

**Reply by TheColonel2688** [reply]
Hi Toni,

Thanks for the reply, I did find that one after I posted here. Currently having issues with it though and waiting on someone to reply to my ticket.

**Comment by amish25** [comment]
Hi do you have a library compatible with Codesys V2.3?
